pulp_python/app/tasks/sync.py

```diff
@@ -140,31 +140,31 @@ def _parse_metadata(project, version, distribution):
 
     package = {}
 
-    package['filename'] = distribution['filename']
-    package['packagetype'] = distribution['packagetype']
-    package['name'] = project['name']
+    package['filename'] = distribution.get('filename') or ""
```

Comment:
I don't think we should ever allow this to be "", since it's a required field

Reply:
You're right but we need a better way to validate that generally.  This patch doesn't actually change that because previously, it was setting the field to ```NULL``` which is also invalid.